components/prism-keepalived.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.keepalived = {
+	'comment': {
+		pattern: /(^|[^"{\\])#.*/,
```

Comment:
Could you please explain the `[^"{\\]` class? Why are these characters not allowed?

Reply:
I don't know too and remove it now. See the above comment.